test/unit/actions-test.js

```diff
@@ -79,6 +81,48 @@ describe('Denali.Action', function() {
     });
   });
 
+  describe.only('an app with a default JSON-API serializer', function() {
+    it('should not invoke the default serializer if serializer=false in an action', function() {
+      let success = false;
+      let mock = mockReqRes({
+        request: {
+          headers: {
+            'content-type': 'application/json',
+            'transfer-encoding': 'chunked',
+          },
+          method: 'post',
+          url: '/a-path',
+          body: { my: 'post-data' }
+        }
+      });
+      class IssuesAction extends Action {
+        serializer = false;
+        run() {
+          success = true;
+          expect(success).to.be.true();
```

Comment:
It's a bit ugly to have this in here, but I **really** didn't want to end up faking all the action methods in here, this took me long enough ...